skore/src/skore/sklearn/train_test_split/train_test_split.py

```diff
@@ -132,32 +132,42 @@ class labels.
     {'X_train': ..., 'X_test': ...,
      'y_train': ..., 'y_test': ...,
      'sample_weights_train': ..., 'sample_weights_test': ...}
+
+    >>> # When using positional arguments and as_dict=True
+    >>> # the first argument is assumed to be X, the second y
+    >>>  train_test_split(
+    ...     [[1], [2], [3], [4]], [0, 1, 0, 1], as_dict=True, random_state=0
```

Comment:
```suggestion
    ...     [[1], [2], [3], [4]], [0, 1, 0, 1], as_dict=True
```
No need for `random_state` since we don't check the output arrays. You can also remove `random_state` in the previous doctest